tests/testsuite/features_namespaced.rs

```diff
@@ -377,7 +377,8 @@ fn no_implicit_feature() {
             r#"
                 fn main() {
                     if cfg!(feature = "regex") { println!("regex"); }
-                    if cfg!(feature = "lazy_static") { println!("lazy_static"); }
+                    // #[expect(unexpected_cfgs)]
+                    // if cfg!(feature = "lazy_static") { println!("lazy_static"); }
```

Comment:
Whats this about?

Reply:
This test is testing that there is no implicit feature for `lazy_implicit`, which is tested by the error message below. This `println!` is therefor useless as a anti-regression since the error-message will fail before it can output anything.

So as far as I can tell this is dead-code, but I didn't want to remove it completely, so it's a just a todo for when `#[expect]` is stabilized (so it can really test something).

I can also just remove it or allow the lint, as you wish.

Reply:
I believe this is trying to show that activating `regex` does not cause a `feature = "lazy_static"` to be created, so I think we should be keeping this.

Reply:
Indeed, that must be it. Reverted that part and put an `#[allow]`.